smoke-tests/apps/SamplingOverrides/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SamplingOverridesThreadName.java

```diff
@@ -50,9 +48,5 @@ static class Tomcat8Java19Test extends SamplingOverridesThreadName {}
   @Environment(TOMCAT_8_JAVA_20)
   static class Tomcat8Java20Test extends SamplingOverridesThreadName {}
 
-  @Environment(WILDFLY_13_JAVA_8)
-  static class Wildfly13Java8Test extends SamplingOverridesThreadName {}
-
-  @Environment(WILDFLY_13_JAVA_8_OPENJ9)
-  static class Wildfly13Java8OpenJ9Test extends SamplingOverridesThreadName {}
+  // intentionally not including wildfly, since this test is set up for Tomcat thread names
```

Comment:
@heyams I'm not sure how this test passed is passing sometimes on wildfly (I couldn't get it to pass locally), but I think ok to just only test on Tomcat